rocketpy/rocket/rocket.py

```diff
@@ -996,11 +1005,17 @@ def __add_single_surface(self, surface, position):
         """Adds a single aerodynamic surface to the rocket. Makes checks for
         rail buttons case, and position type.
         """
-        position = (
-            Vector([0, 0, position])
-            if not isinstance(position, (Vector, tuple, list))
-            else Vector(position)
-        )
+        if isinstance(surface, (TrapezoidalFin, EllipticalFin, FreeFormFin)):
+            # TODO: this depends on cant angle, so it should somehow be
+            # recalculated whenever the cant angle of the fin changes
```

Comment:
maybe what you want to do is to create a setter method to the cant_angle parameter?